assets/js/modules/analytics-4/components/widgets/PopularProductsWidget.js

```diff
@@ -58,9 +62,22 @@ export default function PopularProductsWidget( props ) {
 	);
 
 	const { setValue } = useDispatch( CORE_UI );
+	const [ showTooltip, setShowTooltip ] = useState( true );
 
 	const openMetricsSelectionPanel = useCallback( () => {
+		// Hide the tooltip so it doesn't remain visible, above the panel we're
+		// opening.
+		setShowTooltip( false );
+
+		// Open the panel.
 		setValue( KEY_METRICS_SELECTION_PANEL_OPENED_KEY, true );
+
+		// Wait for the panel to be open before we re-enable the tooltip.
+		// This prevents it from appearing above the slide-out panel, see:
+		// https://github.com/google/site-kit-wp/issues/7060#issuecomment-1664827831
+		setTimeout( () => {
+			setShowTooltip( true );
+		}, 0 );
```

Comment:
Why re-show the tooltip in this case? If the user clicked the link to open the panel, I would think the tooltip should be closed and remain closed in that situation.

Reply:
That's a good point. However, I've kept the tooltip to reappear for the following reasons:

1. The AC didn't specify the behavior. Although, there's a possibility it might have been overlooked.
2. ~In scenarios where users may accidentally close the panel and decide to alter the widgets?~  They can conveniently do so via the `Change Metrics` link.
3. The tooltip reappears upon a page refresh. Hiding it post-panel opening and displaying it again after a refresh might confuse users.

Reply:
It would be better to hide it with a permanent solution if we want to hide it. WDTY? cc: @jimmymadon